packages/ingest-mongodb-public/src/sources/mongodb-university/makeUniversityPages.ts

```diff
@@ -181,3 +207,26 @@ export function convertVideoTranscriptFromSrtToTxt(transcript: string): string {
     .join(" ")
     .trim();
 }
+
+/**
+  Helper function to create Markdown content for MongoDB University Learning Paths and Courses
+  based on titles, duration, and descriptions.
+ */
+export function generateContentDescriptionMarkdown({
+  tiCatalogItem,
+}: {
+  tiCatalogItem: TiCatalogItem;
+}): string {
+  const { name, description, nested_content } = tiCatalogItem;
+  const title = `# ${name}`;
+  const markdownContent = [title, description, `\n`];
```

Comment:
```suggestion
  const markdownContent = title + "\n\n" + description + "\n\n"
```

since you're building a string of, let's keep the data structure as a string while you build it.

this'll make it clearer how to handle the line breaks as well (see above comments).